src/friends/request/request.controller.ts

```
@@ -0,0 +1,16 @@
+import { Controller, Param, Post, Req } from "@nestjs/common";
+import { RequestService } from "./request.service";
+import { Request } from "express";
```

ESLint

[error] 'Request' is defined but never used.

(@typescript-eslint/no-unused-vars)